File: LIP/maior-multiplo-inteiro/C/maior_multiplo_int.c
#include <stdio.h>
#include <limits.h>

#define LIMITE INT_MAX

int main () {
    int num;

    scanf("%d", &num);

    while (num != INT_MAX) {
        num++;
    }

    printf("Maior multiplo: %d", num);

    return 0;
}

File: LIP/maior-multiplo-inteiro/Java/MaiorMultiploInt.java
package Capitulo6;

import java.util.Scanner;

public class MaiorMultiploInt {
	static int INT_MAX = Integer.MAX_VALUE;
	final static Scanner in = new Scanner(System.in);
	
	public static void main(String[] args) {
		System.out.println("");
		int num = in.nextInt();
		
		while (num != INT_MAX) {
			num++;
		}
		
		System.out.println("Maior Multiplo: " + num);
	}
}
